Platform.md

```diff
@@ -74,10 +74,8 @@ This file contains instructions for installing dependencies on different platfor
     
 ### Windows
 
-  * Both the Websocket and OpenSSL builds work on windows. OpenSSL 1.0.2 needs to be installed for them to work properly.
-  * Download and install OpenSSL from [this link](https://slproweb.com/products/Win32OpenSSL.html)
-    * Make sure you download the "full" ~30mb version, the "light" version won't work.
-    * I also needed to create a windows environment variable "OPENSSL_ROOT_DIR" and set it to wherever OpenSSL installed.
+  * Both the Websocket and OpenSSL builds work on windows. OpenSSL 1.0.2o needs to be installed for them to work properly.
```

Comment:
This should be the "latest version of 1.0.2", as the letter releases will keep getting updated.